tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -150,6 +151,18 @@ internal TracerManager CreateTracerManager(ImmutableTracerSettings settings, Tra
             telemetry.RecordProfilerSettings(profiler);
             telemetry.ProductChanged(TelemetryProductType.Profiler, enabled: profiler.Status.IsProfilerReady, error: null);
 
+            var requestedInjectors = settings.PropagationStyleInject;
+            var requestedExtractors = settings.PropagationStyleExtract;
+
+            // If Activity support is enabled, we must enable the W3C Trace Context propagators.
+            // We have not updated the settings object so our configuration telemetry accurately reflects what was set by the user
+            // It's ok to include W3C multiple times, we handle that later.
+            if (settings.IsActivityListenerEnabled)
```

Comment:
I think you forgot to use the local variables, I'll add an extra commit

Reply:
Done in #4465 (and merged in [here](https://github.com/DataDog/dd-trace-dotnet/pull/4460/commits/cfd9a5af279f7cfa47e1e99c69610600480127aa))